src/popup.js

```diff
@@ -125,50 +123,52 @@ function closeOverlay() {
  */
 function send_error(message) {
   var browser = window.navigator.userAgent;
-  var tabId = parseInt($('#associatedTab').attr('data-tab-id'), 10);
```

Comment:
If privacy badger is disabled for a URL, the popup never gets `#associatedTab` set. So here `tabId` ends up as `NaN` and we get sadness. This issue is in master too, but fixed here.